tensorflow/python/kernel_tests/unstack_op_test.py

```diff
@@ -161,6 +161,17 @@ def testZeroLengthDim(self):
       y = array_ops.unstack(x, axis=1)[0].eval()
       self.assertEqual(y.shape, (0, 2))
 
+  def testNumWithTensor(self):
+    dynamic_feed_shape = (2,None)
+    static_feed_shape_dict = {'scalar': (2, ()),'one_dim': (2, [1])}
+    x = array_ops.constant([[1,2,3],[4,5,6]])
+    with self.test_session() as sess:
+      for feed, shape in (dynamic_feed_shape, static_feed_shape_dict['scalar'],static_feed_shape_dict['one_dim']):
```

Comment:
Can you run pyformat or make sure the pylint is happy?

Reply:
I forgot to check pylint.
now, I checked and fixed